R/workflow.R

```diff
@@ -303,6 +310,48 @@ copy_nonmem_model_from <- function(
   return(.new_spec)
 }
 
+
+#####################
+# Result object
+#####################
+
+#' Create a `bbi_nonmem_result` object from a file path (should work with either path to a model file, yaml file, and output folder)
```

Comment:
does this really make sense? like the result object is really the result of _submitting_ the model and contains things like the stdout. Wouldn't it just make sense that the anything that does postprocessing of results can work off of the information in a spec object or path. Basically, any post processing functions that take a result object should be able to act on a spec object or path to yaml (and/or results folder)